mllib-local/src/main/scala/org/apache/spark/ml/impl/Utils.scala

```diff
@@ -18,7 +18,7 @@
 package org.apache.spark.ml.impl
 
 
-private[ml] object Utils {
+private[spark] object Utils {
```

Comment:
If this is the natural home for this, fine, though I'd usually put code used by both in .mllib. Maybe. I'm not sure anymore

Reply:
Yes, I think it is natural to place it in ,ml.
if we keep KMeans in the .mllib side, when we do some improvement, this will happen.